arkouda_starjoin/server/ServerModules.cfg

```diff
@@ -0,0 +1,45 @@
+module StarJoinMsg
+{
+    use ServerConfig;
+
+    use MultiTypeSymEntry;
+    use ServerErrorStrings;
+    use MultiTypeSymbolTable;
+
```

Comment:
I'm not sure why this is in the `ServerModules.cfg`

Reply:
No worries, this shouldn't be in the PR.  I was probably just copy-pasting something else in there and forgot about it.  I haven't actually started on this yet.